src/libs/notification/example/CMakeLists.txt

```diff
@@ -1,17 +1,45 @@
 include (LibAddMacros)
+include (LibAddBinding)
 
-file (GLOB HDR_FILES *.h)
-file (GLOB SRC_FILES *.c)
+## Build example_notification
+set (example example_notification)
 
+SET (SRC_FILES
+  example_notification.c
+)
 add_headers (ELEKTRA_HEADERS)
-set (SOURCES ${SRC_FILES} ${HDR_FILES} ${ELEKTRA_HEADERS})
-
-## Build test
-set (example example_notification)
+set (SOURCES ${SRC_FILES} ${ELEKTRA_HEADERS})
 
 add_executable (${example} ${SOURCES})
 add_dependencies (${example} kdberrors_generated)
 
 # target_include_directories (${example} SYSTEM PUBLIC ${LIBUV_INCLUDE_DIRS})
 
 target_link_elektra (${example} elektra-kdb elektra-notification)
+
+# The asynchronous example requires io_uv
+check_binding_included ("io_uv" IS_INCLUDED SUBDIRECTORY "io/uv" SILENT)
+find_package (PkgConfig)
+pkg_check_modules (LIBUV QUIET libuv>=1)
+if (IS_INCLUDED AND LIBUV_FOUND)
+  ## Build example_notification_async
```

Comment:
spaces?

Reply:
automatic formatting for cmake would be great :)